metagraph-plugin/metagraph_katana/tests/test_import.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import metagraph as mg
+
+import metagraph_katana as mg_katana_src
```

Comment:
Don't use `as` except when it's one of a few well known contractions. The ones I'm OK with are `np` for `numpy` and in this context `mg` for `metagraph`. I don't even like those honestly. But I tolerate them. Don't use others.